fdbclient/include/fdbclient/MetaclusterManagement.actor.h

```diff
@@ -1152,31 +1157,49 @@ struct CreateTenantImpl {
 
 			if (groupEntry.present()) {
 				ASSERT(groupEntry.get().assignedCluster.present());
+				if (self->preferAssignedCluster &&
+				    groupEntry.get().assignedCluster.get() != self->tenantEntry.assignedCluster.get()) {
+					throw invalid_tenant_configuration();
```

Comment:
Let's add a trace event here so that we can elaborate on the fact that the tenant group and assigned clusters did not match as well as what the assigned cluster for the tenant group was.